css/styles.css

```diff
@@ -111,7 +117,8 @@
     width: 40px;
 }
 .ab-column .ab-medium-field {
-    width: 225px;
+    width: 100%;
+    max-width :285px;
```

Comment:
Yes, you might _like_ the previous better. I _like_ this one. Both are opinions. But one might argue, that we can use the whitespace we have, to allow a textarea with a little more space for the user to write in. 
The max-width / width combination seems a bit redundant, but it allows the given field to shrink if the viewport is smaller than our fixed with container.